docs/concepts/work-queues.md

```diff
@@ -49,6 +49,7 @@ Configuration parameters you can specify when starting an agent include:
 | --run-once        | Only run agent polling once. By default, the agent runs forever. |
 | --prefetch-seconds | The amount of time before a flow run's scheduled start time to begin submission. Default is the value of `PREFECT_AGENT_PREFETCH_SECONDS`. |
 | --hide-welcome   | Do not display the startup ASCII art for the agent process.                          |
+| --match | Start all queues that match the given match string
```

Comment:
```suggestion
| --match | Start all queues that match the given match string. |
```

Reply:
Let's make clear here that this param is about matching prefixes, and will ignore queues passed with `-q`.
How about: 
| -- match QUEUE_PREFIX | Polls for queues matching the given QUEUE_PREFIX. Note: this option overrides anything passed with `-q` (or `--work-queue`)

Reply:
The doc format above with `QUEUE_PREFIX` is not that consistent, with only the `--api TEXT` being in this form.. presumably to save space.   (-q and --prefetch-seconds could include the param_name). Just a note for future PRs

Reply:
If it's going to cause `-q` to be ignored we should throw an error if you try to use both.

Reply:
That's much better / safer!

Reply:
Sorry I was wrong, I read downstream code to get that behavior. But it actually already prevents you from passing both, as @madkinsz suggested:
>"Only one of `work_queues`, `match`, or `tags` can be provided."

gets printed if the user attempts to pass more than 1. (you can remove the `Note:` here)